core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java

```diff
@@ -53,14 +54,22 @@ public AbstractComplianceChecker() {
         try {
             execute(evaluation);
         } catch (FailCausingException e) {
-            // TODO: A FailCausingException should not end the build but should be reflected in the workflow result
-            // Build should not be aborted, but should indicate the failure in the end, so that reports etc. are created.
-            throw new ExecutionException("Fail execution because of compliance check result", e);
+            // no-op
+            // failCausingResults were added to evaluationResults already
         }
         LOGGER.info("Check evaluation results... done.");
         return artifacts;
     }
 
+    @Override
+    public WorkflowStepResult postProcessResult(WorkflowStepResult result) {
+        WorkflowStepResult pResult = super.postProcessResult(result);
+        if (iEvaluationResults.size() > 0) {
```

Comment:
The I in IEvaluationResult is a sin of the past, please stop with sins by using the i in variable names 👎